src/angularjs/src/app/users/detail/users-detail.controller.js

```diff
@@ -100,9 +99,18 @@
                     else {
                         toastr.info('User deactivated');
                     }
+                }, function(error) {
+                    displayError(error);
                 });
             }
         }
+
+        // Helper to display error popup on DRF request failure
+        function displayError(error) {
+            if (error.data && error.data.detail) {
+                toastr.error(error.data.detail, {timeOut: 5000});
+            }
```

Comment:
For better error visibility, an else block that calls `$log.error(error)` might be useful